Objects/Converters/ConverterRhinoGh/ConverterRhinoGhShared/BrepEncoder.cs

```diff
@@ -133,7 +133,7 @@ private static BrepIssues AuditBrep(Brep brep, double modelRelativeTolerance)
           }
 
           kinkyEdges += edges.Count - edgeCount;
-#if RHINO7
+#if RHINO7 || RHINO8
```

Comment:
Should we start thinking of adding `RHINO7_OR_GREATER` flags to prevent this?

It may become more manageable and it's an already existing pattern in the tooling (i.e. `NET47_OR_GREATER`)
![Screenshot 2023-12-11 at 10 16 31](https://github.com/specklesystems/speckle-sharp/assets/2316535/53269d94-14a8-467d-8178-2c1e505f2f48)

Reply:
Yes lets start doing this please :)

Reply:
Done 🥳